pkg/gui/controllers/status_controller.go

```diff
@@ -198,6 +187,28 @@ func (self *StatusController) showAllBranchLogs() error {
 	})
 }
 
+func (self *StatusController) showDashboard() error {
+	dashboardString := strings.Join(
+		[]string{
+			lazygitTitle(),
+			"Copyright 2022 Jesse Duffield",
```

Comment:
Should we update the copyright to display a range of years?

Reply:
We can just use `time.Now().Year()`

Reply:
Changed to:
`fmt.Sprintf("Copyright %d Jesse Duffield", time.Now().Year())`